mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -242,7 +299,7 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		}
 
 		// optional
-		pnh.param("covariance", p->covariance, 0);
+		if (pnh.getParam("covariance", p->covariance) >= 0) p->cov_is_def = true;
```

Comment:
In previous comments i mean that `p->covariance` <= 0 meaning that it is not defined, so
no need for `cov_is_def` flag.